specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json

```diff
@@ -0,0 +1,299 @@
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {    
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ddosCustomPolicies/{ddosCustomPolicyName}": {      
+      "delete": {
+        "tags": [
+          "ddosCustomPolicies"
+        ],
+        "operationId": "DdosCustomPolicies_Delete",
+        "description": "Deletes the specified DDoS custom policy.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ddosCustomPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the DDoS custom policy."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "200": {
+            "description": "Delete successful."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete DDoS custom policy": {
+            "$ref": "./examples/DdosCustomPolicyDelete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ddosCustomPolicies"
+        ],
+        "operationId": "DdosCustomPolicies_Get",
+        "description": "Gets information about the specified DDoS custom policy.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ddosCustomPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the DDoS custom policy."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
"default" response missing for all APis. The schema for the "default" response should match the ARM error contract. Check Batch RP swagger for a sample.

Reply:
default responses added.